app/api/routes.py

```
@@ -540,4 +542,28 @@ def get_consult_history_detail_routes(
+@router.get(
+    "/api/v1/stats/weight-uf-trend",
+    tags=["통계"],
+    summary="최근 7일간 체중 및 일별 제수량 추이 조회",
+    description=(
+        "최근 7일 동안의 체중과 제수량 추이를 반환합니다."
+        "날짜별 기록이 없는 경우 해당 일자는 weight/total_uf가 null로 반환됩니다."
+    ),
+    response_model=List[WeightUfPoint],
+)
+def get_weight_uf_trend(
+    db: Session = Depends(get_db),
```

Ruff (0.14.5)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)